R/missingDataImputation.R

```diff
@@ -219,20 +219,30 @@ MissingDataImputation <- function(jaspResults, dataset, options) {
     predMat <- passive$pred
   }
 
-  miceOut <- try(
-    with(options,
-      mice::mice(
-        data            = dataset,
-        m               = nImp,
-        method          = methVec,
-        predictorMatrix = predMat,
-        visitSequence   = visitSequence,
-        maxit           = nIter,
-        seed            = seed,
-        print           = FALSE
-      )
+  if (!is.null(jaspResults[["MiceMids"]]$object)) {                     # if we already computed some iterations
```

Comment:
This function doesn't know about `jaspResults[["MiceMids"]]`. The analogous function argument is called `miceMids`. 

Replacing all instances of `jaspResults[["MiceMids"]]` with `miceMids` should solve the problem.